lib/librato/collector/aggregator.rb

```diff
@@ -26,13 +29,13 @@ def [](key)
       def fetch(key, options={})
         return nil if @cache.empty?
         return fetch_percentile(key, options) if options[:percentile]
-        gauges = nil
-        source = options[:source]
-        @lock.synchronize { gauges = @cache.queued[:gauges] }
-        gauges.each do |metric|
+        measurements = nil
+        tags = options[:tags]
+        @lock.synchronize { measurements = @cache.queued[:measurements] }
+        measurements.each do |metric|
```

Comment:
same comment as on lib-metrics, this feels odd to me since metrics contain series of measurements but this code reads as `measurements` being a set of metrics.